auth-web/src/views/management/UserManagement.vue

```diff
@@ -203,7 +202,7 @@ export default class UserManagement extends Vue {
   private readonly leaveTeam!: (memberId: number) => void
   private readonly syncOrganizations!: () => Promise<Organization[]>
 
-  private notifyUser = false
+  private notifyUser = true
```

Comment:
We will need some cleanup of unused flags, unused text, etc.  That can be done separately from this I think though.